src/main/java/com/blockout/Log.java

```diff
@@ -0,0 +1,29 @@
+package com.blockout;
+
+import com.minecolonies.lib.Constants;
```

Comment:
Yeah, true that. I could inline the constant, but we should prepoend our blockout loggings with Minecolonies so that people reading the mod go to us and not to lunatics